src/QMCWaveFunctions/Jastrow/BsplineFunctor.cpp

```diff
@@ -7,7 +7,7 @@
 // File developed by: Ken Esler, kpesler@gmail.com, University of Illinois at Urbana-Champaign
 //                    Jeongnim Kim, jeongnim.kim@gmail.com, University of Illinois at Urbana-Champaign
 //                    Jeremy McMinnis, jmcminis@gmail.com, University of Illinois at Urbana-Champaign
-//
+//                    
```

Comment:
Fashion.

Reply:
Nobody has ever accused me of being fashionable before.  How do I accept your suggestion?

Reply:
Sadly, I feel PR owners didn't self review their PR first and made an effort to reduce the amount diffs. I really wish I could put less comment.

Reply:
I did comb over this in what I thought was a thorough fashion.  What I have learned is that using the intel compiler in a development situation is a bad idea.  For instance, the reason the tests have been failing is that I had forgot to return the success variable at the end of the put method for the RadialJastrowBuilder class.  However, on intel the unit test still passed.

Anyway, I'm new to the formal development process here.  Let me know if it would be preferable for me to make a clean new pull request, or just fix these few things.

Reply:
I wouldn't say doing development with Intel is that bad. At least you can cover two compilers.
Just fix the current branch. No need for a new PR.